From linux-usb-owner@vger.kernel.org Mon May  5 00:33:35 2008
From: Sebastian Siewior <linux-usb@ml.breakpoint.cc>
Date: Mon, 5 May 2008 09:31:50 +0200
Subject: usb: fix compile warning in isp1760
To: Greg Kroah-Hartman <gregkh@suse.de>
Cc: linux-usb@vger.kernel.org
Message-ID: <20080505073150.GA21003@Chamillionaire.breakpoint.cc>
Content-Disposition: inline


From: Sebastian Siewior <bigeasy@linutronix.de>

drivers/usb/host/isp1760-if.c:275: warning: 'ret' is used uninitialized in this function

Signed-off-by: Sebastian Siewior <bigeasy@linutronix.de>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/host/isp1760-if.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/usb/host/isp1760-if.c
+++ b/drivers/usb/host/isp1760-if.c
@@ -256,7 +256,7 @@ static struct pci_driver isp1761_pci_dri
 
 static int __init isp1760_init(void)
 {
-	int ret;
+	int ret = -ENODEV;
 
 	init_kmem_once();
 
